event/include/Event.h

```diff
@@ -85,7 +85,10 @@ class Event : public IEvent {
 
         /** @return Get a mutable copy of the EventHeader. */
         EventHeader& getEventHeaderMutable() const { return *event_header_; }
-
+	
+	//TODO document
```

Comment:
Why save for later and not just document?

Reply:
Hi Maurik, 

I have bunch of //TODO here and there in the code. Perhaps I should decide to open an issue and fix all of them. I'll do it.